content/04. box_1_experimental_design.md

```diff
@@ -0,0 +1,40 @@
+#### Box 1: Understanding experimental design of machine learning to inform requirements for data
+
+##### Components of ML experiments
+
+Machine learning algorithms identify patterns that explain or fit a given dataset.
+Every machine learning algorithm (_model_) goes through a _training_ phase where it identifies underlying patterns in a given dataset, and a _testing_ phase where the model applies the identified patterns to unseen data points.
+So every model should be provided with the following fundamental parts as input: 1. a _training dataset_ , 2. an _evaluation dataset_ , 3. a _held-out validation dataset_ .
+Typically each of these input datasets consists of a matrix (or dataframe or table) where samples form the rows and features form the columns.
```

Comment:
```suggestion
These input data can be images, text, numbers, or other types of data which are typically encoded as a numerical representation of the input data. 
```

I think this is safe to say is typically true, but there are probably exceptions....